src/app/Dashboard/styles/dashboard.scss

```diff
@@ -3,6 +3,56 @@
   justify-content: space-around;
 }
 
+.pagination {
+  justify-content: center;
+}
+
+.pagination > li {
+  display: inline-block;
+  padding-left: 0;
+}
+
+.pagination > li {
```

Comment:
The selector `.pagination > li` has already been defined in line 10

Reply:
@Chris-Maina I have fixed this